crates/swc_ecma_parser/src/error.rs

```diff
@@ -83,7 +83,7 @@ pub enum SyntaxError {
     UnterminatedStrLit,
     ExpectedUnicodeEscape,
     EscapeInReservedWord {
-        word: JsWord,
+        word: Word,
```

Comment:
This change makes this PR a breaking change, so I think it's better to revert this.